src/git.ts

```diff
@@ -37,13 +37,15 @@ async function capture(cmd: string, args: string[]): Promise<ExecResult> {
 
 export async function cmd(...args: string[]): Promise<string> {
     core.debug(`Executing Git: ${args.join(' ')}`);
+    const { owner } = github.context.repo;
+    const serverUrl = github.context.payload.repository?.html_url?.split(`/${owner}`)[0];
```

Comment:
this logic is duplicated in a couple of places. I suggest extracting it into 2 functions and reusing those.
```ts
export function getServerUrl(repositoryUrl: string): string {
  return github.context.payload.repository?.html_url?.split(`/${owner}`)[0];
}

export function getServerName(repositoryUrl: string): string {
  return getServerUrl(repositoryUrl).split('//')[1]
}

```

Reply:
Done [here](https://github.com/mjochum/github-action-benchmark/blob/fix/serverUrls/src/git.ts#L40)